conman/routes/models.py

```diff
@@ -45,8 +46,9 @@ def check(cls):
 
     def get_ancestors(self):
         """Get all the ancestors of this Route."""
-        paths = split_path(self.url)[:-1]
+        assert self.pk  # Ensure object has been saved.
```

Comment:
By the looks of it, [it can be explicitly set to `0`](https://github.com/django/django/commit/d3cf6cfacfb828faad4f4f97c904e259304649b3)!

I have no idea why someone would want to do that, but I'll update this to deal with it. Good catch!